plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/handler/ResultSetPropertyTester.java

```diff
@@ -106,7 +107,7 @@ private boolean checkResultSetProperty(ResultSetViewer rsv, String property, Obj
                 }
                 if ("edit".equals(expectedValue) || "inline".equals(expectedValue)) {
                     DBDAttributeBinding attr = rsv.getActivePresentation().getCurrentAttribute();
-                    if (attr == null) {
+                    if (attr == null || !(rsv.getActivePresentation() instanceof IResultSetEditor)) {
```

Comment:
That's a really dirty hack.

Use `DBSDataContainer#getSupportedFeatures` and exclude `FEATURE_DATA_FILTER` for the <kbd>Statistics</kbd> container.